crates/holochain/src/core/ribosome/real_ribosome.rs

```diff
@@ -118,6 +117,8 @@ use wasmer_middlewares::metering::get_remaining_points;
 use wasmer_middlewares::metering::set_remaining_points;
 use wasmer_middlewares::metering::MeteringPoints;
 
+pub type ModuleCacheLock = parking_lot::RwLock<ModuleCache>;
```

Comment:
I made an alias for this because I was experimenting with making this a `tokio::sync::RwLock` and wanted to more quickly change between the two.